src/LocalNeuralNet.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef LOCALNEURALNET_HPP_INCLUDE
+#define LOCALNEURALNET_HPP_INCLUDE
+
+#include <memory>
+#include <vector>
+
+namespace geopm
+{
+    ///  @brief Class to manage data and operations of feed forward neural nets
+    ///         required for neural net inference.
+
+    class LocalNeuralNet
+    {
+        public:
+            /// @brief Returns a unique_ptr to a concrete object
+            ///        constructed using the underlying implementation
+            ///        from an array of doubles of weights and biases.
```

Comment:
Minor discrepancy: this comment talks about doubles, while the actual code uses floats.

Reply:
I meant pairs!